website/layouts/partials/_default/sidemenu.html

```diff
@@ -1,68 +1,149 @@
 {{ $currentPage := . }}
```

Comment:
I guess we can put this in `versioning` dir too

Reply:
I think it's better for it to remain in default since it serves as our default sidemenu of choice 😁